include/event2/util.h

```diff
@@ -717,6 +717,12 @@ int evutil_ascii_strcasecmp(const char *str1, const char *str2);
 EVENT2_EXPORT_SYMBOL
 int evutil_ascii_strncasecmp(const char *str1, const char *str2, size_t n);
 
+/** As strcasestr, but always searching substring in locale-independent
+    ASCII.  That's useful if you're handling data in ASCII-based protocols.
+ */
+EVENT2_EXPORT_SYMBOL
+const char* evutil_ascii_strcasestr(const char* s, const char *find);
```

Comment:
Let's not export this function

Reply:
Fixed

Reply:
OK, let's move it into `util-internal.h`, this will be enough. Done